problem2.cs

```diff
@@ -0,0 +1,52 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    /*
+     * Complete the 'simpleArraySum' function below.
+     *
+     * The function is expected to return an INTEGER.
+     * The function accepts INTEGER_ARRAY ar as parameter.
+     */
+
+    public static int simpleArraySum(List<int> ar)
+    {int i,sum=0;
```

Comment:
declare i directly in for loop

Reply:
instead of int for sum, you can use the var keyword. Using var keyword instead of explicit data types such as int is considered as best practice

Reply:
@narayanagunturi done all the changes commented, plz check and comment if i have to do any other changes.